API.md

```diff
@@ -269,6 +269,12 @@ An `HTTP 404 Not Found` status code is returned if the case with the specified u
 ## List Case Events for Case
 * `GET /cases/{caseId}/events` will return a list of case events for the case with given case ID.
 
+&mdash;an optional `category` string query parameter can be used to specify that the JSON response include matching events with the case for said case id. For example:
+
+* `GET /cases/{caseId}/events?category=OFFLINE_RESPONSE_PROCESSED`
+
+If this query parameter is omitted all events for said case ID will be returned.
+
```

Comment:
"In this scenario all events for a given case ID can be retrieved by using the `GET /cases/{caseId}/events` endpoint"

Might want to add this scenario example like how we do it for the rest of the `API.md`, wording is up to you.

Reply:
I think its pretty explicit tbh

Reply:
It is just how its done throughout the rest of the file etc.